.github/workflows/nightly.yml

```diff
@@ -254,6 +254,13 @@ jobs:
           docker rmi test-image
           rm -rf app/payara
 
+      - name: Run Archetype for EE 8, SE 8, TomEE
+        run: |
+          mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -DjavaVersion=8 -Druntime=tomee -DoutputDirectory=app/tomee -Dgoals="clean package" | tee mvn_output.txt
+          
+          MAVEN_EXIT_CODE=${PIPESTATUS[0]}
+          ERROR_MESSAGE="TomEE requires Web Profile and Jakarta EE 8 or 9.1."
```

Comment:
The error messages you need to check for will be from the Archetype, not the UI. Please look here: https://github.com/eclipse-ee4j/starter/blob/master/archetype/src/main/resources/META-INF/archetype-post-generate.groovy.